src/SmartComponents/modules/reducers.js

```diff
@@ -11,11 +11,19 @@ const initialState = {
     baselines: [],
     historicalProfiles: [],
     previousStateSystems: [],
-    stateFilters: [
-        { filter: 'SAME', display: 'Same', selected: true },
-        { filter: 'DIFFERENT', display: 'Different', selected: true },
-        { filter: 'INCOMPLETE_DATA', display: 'Incomplete data', selected: true }
-    ],
+    if (insights.chrome.isBeta()) {
+        stateFilters: [
+            { filter: 'SAME', display: 'Expected', selected: true },
+            { filter: 'DIFFERENT', display: 'Unexpected', selected: true },
+            { filter: 'INCOMPLETE_DATA', display: 'Uncertain', selected: true }
+        ]
+    } else {
+        stateFilters: [
+            { filter: 'SAME', display: 'Same', selected: true },
+            { filter: 'DIFFERENT', display: 'Different', selected: true },
+            { filter: 'INCOMPLETE_DATA', display: 'Incomplete data', selected: true }
+        ]
+    },
```

Comment:
This if won't work inside the hash declaration. You'll want to move it after wards. So having the hash have sane defaults (the way it was prior to the change) and then modifying, e.g.

```
const initialState = {
    fullCompareData: [],
    filteredCompareData: [],
    sortedFilteredFacts: [],
    referenceId: undefined,
    systems: [],
    baselines: [],
    historicalProfiles: [],
    previousStateSystems: [],
    stateFilters: [
            { filter: 'SAME', display: 'Same', selected: true },
            { filter: 'DIFFERENT', display: 'Different', selected: true },
            { filter: 'INCOMPLETE_DATA', display: 'Incomplete data', selected: true }
    ],
    factFilter: '',
    totalFacts: 0,
    page: 1,
    factSort: ASC,
    stateSort: DESC,
    perPage: 50,
    loading: false,
    expandedRows: [],
    error: {},
    emptyState: true
};

if (insights.chrome.isBeta()) {
        initialState['stateFilters'] = [
            { filter: 'SAME', display: 'Expected', selected: true },
            { filter: 'DIFFERENT', display: 'Unexpected', selected: true },
            { filter: 'INCOMPLETE_DATA', display: 'Uncertain', selected: true }
        ]
};

```